tutorials/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -620,7 +620,29 @@ Naming conventions
 
 These naming conventions follow the Godot Engine style. Breaking these will make
 your code clash with the built-in naming conventions, leading to inconsistent
-code.
+code. As a summary table:
+
++---------------+----------------+----------------------------------------------------+
+| Type          | Convention     | Example                                            |
++===============+================+====================================================+
+| File names    | snake_case     | ``yaml_parser.gd``                                 |
++---------------+----------------+----------------------------------------------------+
+| Class names   | PascalCase     | ``class_name YAMLParser``                          |
++---------------+----------------+----------------------------------------------------+
+| Node names    | PascalCase     | ``const Weapon = preload("res://weapon.gd")``      |
```

Comment:
Meh. Hold on, this is misleading. Yes, by convention it's PascalCase for Node names, but the example shows a Script loaded as a constant. It's entirely different.

Reply:
I don't know how I missed that...